softlayer/resource_softlayer_virtual_guest.go

```diff
@@ -615,6 +620,12 @@ func resourceSoftLayerVirtualGuestCreate(d *schema.ResourceData, meta interface{
 		return err
 	}
 
+	// Set notes
+	err = setNotes(id, d, meta)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Why can't the notes be set as part of the submission of the rest of the virtual guest data structure (order template above)?